.github/workflows/go.yml

```diff
@@ -11,13 +11,19 @@ jobs:
   build:
```

Comment:
I think move all commits from https://github.com/aayushduwadi/aeron-go/pull/3 into this PR and update this's PRs title.